README.md

```diff
@@ -73,3 +73,5 @@ Some portions of the codebase are derived from other projects under different li
 ## Known issues
 
 Due to a bug in the Linux kernel, using rkt's overlay support on top of an overlay filesystem requires Linux 4.3+.
```

Comment:
(e.g. as an end user I'd find "In some configurations" a little frustratingly vague)